ROMFS/px4fmu_common/init.d-posix/2507_cloudship

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
+# @name Cloudship
+# @type Airship
+# @class Airship
+#
+# @output MAIN1 stabilator
+# @output MAIN2 motor 1 starboard
+# @output MAIN3 motor 2 port
+# @output MAIN4 rudder motor
+
+sh /etc/init.d/rc.airship_defaults
+
+param set PWM_MAX 2000
+param set PWM_MIN 1000
+
+param set MPC_MANTHR_MIN 0.00
+param set MPC_THR_MIN 0.00
+param set MPC_THR_CURVE 1
+param set MPC_THR_HOVER 0.0
```

Comment:
Why is this 0?

Reply:
Params useful for velocity control in future, but I've removed them for now.